DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/javascriptIndexes/javascriptMapIndex/javascriptMapIndex.java

```diff
@@ -0,0 +1,56 @@
+
+
+import net.ravendb.client.documents.indexes.AbstractJavaScriptIndexCreationTask;
+import net.ravendb.client.documents.session.IDocumentSession;
+
+import net.ravendb.demo.common.DocumentStoreHolder;
+import java.util.List;
+import com.google.common.collect.Sets;
+
+public class javascriptMapIndex {
+    //region Demo
+    //region Step_1
+    public class Employees_ByImportantDetailsJS extends AbstractJavaScriptIndexCreationTask {
+    //endregion
+        //region Step_2
+        public class IndexEntry {
```

Comment:
https://github.com/ravendb/demo/blob/master/DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/mapReduceIndex/MapReduceIndex.java#L32
let's follow same logic - getters/setters - lower case for props

Reply:
Done

Reply:
why we don't have any private/public here?